terraform/modules/ops/main.tf

```diff
@@ -79,18 +65,12 @@ resource "google_artifact_registry_repository" "e2e_runner_docker" {
   ]
 }
 
-###
-# Secret Manager
-###
+###########################
+# End-user Authentication #
+###########################
 
-# OAuth 2.0 secrets
-# These secret resources are REQUIRED, but configuring them is OPTIONAL.
```

Comment:
Ditto - is there a reason we're deleting these comments?

Reply:
I did some comment refactoring, but I think I lost something. Will fix.